pkg/plugin/view/kubehunter/report_view.go

```diff
@@ -74,3 +80,25 @@ func createTable(section starboard.KubeHunterOutput) component.Component {
 
 	return table
 }
+
+func NewKubeHunterReportSummary(report *starboard.KubeHunterReport) (summary *component.Summary) {
+	totalHigh := 0
+	totalMedium := 0
+	totalLow := 0
+
+	for _, section := range report.Report.Vulnerabilities {
+		totalHigh += map[bool]int{true: 1, false: 0}[section.Severity == "high"]
```

Comment:
I'd do that with switch statement to make this code a bit more readable and avoid creating maps in the loop